embulk-core/src/test/java/org/embulk/jruby/TestJRubyInitializer.java

```diff
@@ -3,31 +3,21 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import java.util.ArrayList;
-import java.util.List;
 import org.junit.Test;
 import org.slf4j.LoggerFactory;
 
 public class TestJRubyInitializer {
     @Test
     public void testArguments() {
-        final List<String> loadPath = new ArrayList<>();
-        loadPath.add("/load/path");
-        final List<String> classpath = new ArrayList<>();
-        classpath.add("/classpath");
-        final List<String> options = new ArrayList<>();
-        options.add("--option");
-        options.add("arg");
-
         final JRubyInitializer initializer = JRubyInitializer.of(
                 null,
                 LoggerFactory.getLogger(TestJRubyInitializer.class),
                 "/gem/home",
                 null,
                 true,
-                loadPath,
-                classpath,
-                options,
+                "/load/path",
+                "/classpath",
```

Comment:
these paths are with separator, I would add an additional test (if you don't consider it as a separate refactoring etc.)

Reply:
Thanks for the good catch! Sounds reasonable. In the same test case `testArguments`, added another path for `load_path` and `classpath` in ee06c77b752451afb7ce2028bf85fd109a046114 to test separation by the path separator.